components/breadcrumb/overview.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Overview
+page_title: Breadcrumb Overview
+description: Overview of the Breadcrumb for Blazor.
+slug: breadcrumb-overview
+tags: telerik,blazor,breadcrumb,overview
+published: True
+position: 0
+---
+
+# Breadcrumb Overview
+
+The <a href="https://www.telerik.com/blazor-ui/breadcrumb" target="_blank">Blazor Breadcrumb component</a> allows navigation within a folder structure or web page. It provides an easy way to navigate backwards by one or multiple steps. In addition to built-in [navigation capabilities]({%slug breadcrumb-navigation%}), you can browse through the items, define [templates]({%slug breadcrumb-templates%}) for the individual nodes, render text and [icons/images]({%slug breadcrumb-icons%}), and respond to [events]({%slug breadcrumb-events%}).
+
+>caption In this article:
+
+* [Basics](#basics)
+* [Features](#features)
+
+
+## Basics
+
+To use a Telerik Breadcrumb for Blazor:
+
+1. add the `TelerikBreadcrumb` tag
+1. provide a collection of models to its `Data` property (read more in the [Data Binding article]({%slug breadcrumb-data-binding%}))
```

Comment:
I may be wrong, but "model" (singular) signifies 1) the object type or 2) a collection of data for a page or component (https://en.wikipedia.org/wiki/Data_model). So using the word in plural may be technically incorrect.

```suggestion
1. provide a collection of objects to its `Data` property (read more in the [Data Binding article]({%slug breadcrumb-data-binding%}))
```